src/components/PreviewFrame.jsx

```diff
@@ -9,6 +9,7 @@ import {sourceDelimiter} from '../util/generatePreview';
 const sandboxOptions = [
   'allow-forms',
   'allow-popups',
+  'allow-same-origin',
```

Comment:
@outoftime that makes sense, I didn't consider the case of opening someone else's project. Your proposal to attack the issue from the other side of the iframe has actually given me what I think is an even smoother solution - just add/remove `pointer-events: none` from the iframe's `style` on the divider's `onMouseUp`/`onMouseDown`! I'm going to roll with that.